src/tests/suite/daos_obj.c

```diff
@@ -3078,6 +3079,12 @@ tgt_idx_change_retry(void **state)
 	int			 i;
 	int			 rc;
 
+	dt_redun_fac = DAOS_PROP_CO_REDUN_RF1;
```

Comment:
We can save the old dt_redun_fac and restore it after using.

Reply:
sure, that's better, although now always set it before use. will change it if need to refresh it or on following patch later. thx